### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by Soft-Academy-Course-Administrator**
Modifying readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # git-introduction
 teaching git to new course students.
+some other changes
```
Comment (bozhidar-dimitrov) [inline README.md]: This is unnecessary. Please remove it

Comment (bozhidar-dimitrov) [inline README.md]: Bad line

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # git-introduction
 teaching git to new course students.
```
Comment (bozhidar-dimitrov) [inline README.md:2]: Another line

File: README.md (hunk 1 above)
Comment (Soft-Academy-Course-Administrator) [inline README.md]: Yes probably

**Comment by Soft-Academy-Course-Administrator** [comment]
File reverted

**Comment by bozhidar-dimitrov** [comment]
Well done !

**Pull request merged**